docs/sphinx/user_guide/feature/view.rst

```diff
@@ -330,6 +330,55 @@ indices by :math:`N`, ::
     }
   }
 
+Index Layout
+^^^^^^^^^^^^^  
+``RAJA::IndexLayout`` is a layout that uses an index list to access entries of a 
+view.  This layout requires an indexing strategy for each dimension, which 
+determines how to map a given input index to an entry within a view.

```

Comment:
It seems like the second sentence contradicts the first. Is this a layout that uses index lists or one with options in each dimension?

Reply:
Good point.  Fixed.